samples/rendezvous/src/show-user-profile/show-user-profile.html

```diff
@@ -56,6 +56,10 @@ <h4 id="username-text">
                 </td>
               </tr>
             </table>
+            <dv.tx>
```

Comment:
Based on our conversation, I created a new `dv.button-link` component to use in cases like this. See

https://github.com/spderosso/deja-vu/blob/master/packages/core/src/app/dv/button-link/button-link.component.ts

Reply:
That's great!! I changed it but I think I need to merge to master before checking it is correct. I'm pretty confident on what I wrote so I will just commit it. :)

Reply:
you can merge `master` to this branch. I'll still see only your changes that are not in `master` yet

Reply:
I had some trouble running it after I merged master. Let me write an email explaining that to you!